lib/g_searcher_web/templates/layout/app.html.eex

```diff
@@ -13,10 +13,16 @@
       <section class="container">
         <nav role="navigation">
           <ul>
-            <li><a href="https://hexdocs.pm/phoenix/overview.html">Get Started</a></li>
             <%= if function_exported?(Routes, :live_dashboard_path, 2) do %>
               <li><%= link "LiveDashboard", to: Routes.live_dashboard_path(@conn, :home) %></li>
             <% end %>
+
+            <%= if @conn.assigns.user_signed_in? do %>
```

Comment:
For now, we could go with this but later I prefer to move this into a view method, like `user_singed_in? or signed_in?`, what do you think @bterone

Reply:
Yeah, I agree. 👍